server/interceptor/interceptor.go

```diff
@@ -437,8 +439,14 @@ func newRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
 		if info.readonly {
 			kind = quota.Read
 		}
+
+		info.quotaUsers = quotaUser
 		for _, user := range chargedUsers(req) {
 			info.specs = append(info.specs, quota.Spec{Group: quota.User, Kind: kind, User: user})
+			if len(info.quotaUsers) > 0 {
```

Comment:
Does this list need to be canonicalised (e.g. sorted) before joining?

Reply:
I think not, given that the users are added in a specific order in the first place.